### PR TITLE
Improve error message when when creating an emulator and no system image is found

### PR DESCRIPTION
**Pull request by guidezpl**

### CODE REVIEW

File: packages/flutter_tools/lib/src/emulator.dart [hunk 1]
```diff
@@ -126,7 +126,8 @@ class EmulatorManager {
           error:
               'No suitable Android AVD system images are available. You may need to install these'
               ' using sdkmanager, for example:\n'
-              '  sdkmanager "system-images;android-27;google_apis_playstore;x86"');
+              ' sdkmanager "system-images;android-33;google_apis_playstore;x86_64"\n'
+              ' See all available images with sdkmanager --list.');
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/emulator.dart:130]: did you intend to slightly un-indent the code snippet and indent the last line?

convention is to have the spaces on the right hand side, which would make the actual indentation more obvious.

**Comment by christopherfujino** [comment]
Thank you for contributing! As you have not responded within 3 weeks, I will be closing this PR as stale. If you are still interested in landing this, feel free to rebase this PR upstream, address code review feedback, and re-open this PR.

File: packages/flutter_tools/lib/src/emulator.dart (hunk 1 above)
Comment (guidezpl) [reply packages/flutter_tools/lib/src/emulator.dart:130]: Fixed

**Comment by christopherfujino** [comment]
> It looks like this pull request may not have tests. Please make sure to add tests before merging. If you need [an exemption](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests) to this rule, contact Hixie on the #hackers channel in [Chat](https://github.com/flutter/flutter/wiki/Chat) (don't just cc him here, he won't see it! _He's on Discord!_).
> 
> If you are not sure if you need tests, consider this rule of thumb: the purpose of a test is to make sure someone doesn't accidentally revert the fix. Ask yourself, **is there anything in your PR that you feel it is important we not accidentally revert back to how it was before your fix?**
> 
> **Reviewers**: Read the [Tree Hygiene page](https://github.com/flutter/flutter/wiki/Tree-hygiene#how-to-review-code) and make sure this patch meets those guidelines before LGTMing.

This change looks reasonable; per the bot this will either need a unit test or an exemption.

**Comment by Hixie** [comment]
ideally that should be tested, if only to prove we intended the indentation to be a particular way

**Comment by guidezpl** [comment]
Unfortunately, I don't have time to dig into how `flutter_tool` tests things, so I'll close this.

**Pull request closed without merging**